pyproject.toml

```diff
@@ -22,12 +22,12 @@ classifiers = [
     "Topic :: Scientific/Engineering",
     "Topic :: Utilities",
 ]
-dependencies = ["bpack[bs,np]", "numpy", "tqdm"]
+dependencies = ["bpack[bs,np]", "numpy", "tqdm","bitstruct","bpack"]
 dynamic = ["version"]
 
 
 [project.optional-dependencies]
-cli = ["tqdm", "argcomplete"]
+cli = ["tqdm", "argcomplete","pandas","polars","hdf5"]
```

Comment:
I would prefer to split the cli requirements form hdf5:

```diff
- cli = ["tqdm", "argcomplete","pandas","polars","hdf5"]
+ cli = ["tqdm", "argcomplete"]
+ hdf5 = ["pandas[hdf5]"]
+ polars = ["polars"]
```

and to install them all one could use:

```
$ python3 -m pip install s1isp[cli,hdf5,polars]
```

Reply:
Updated the toml and readme file to reflect this aspect